argyrogounari/assignment3/Dictionary.java

```diff
@@ -0,0 +1,35 @@
+/**
+* We were given a dictionary class with these two methods:
+* isWord(string): Returns whether the given string is a valid word.
+* isPrefix(string): Returns whether the given string is a prefix of at least one word in the
+* dictionary.
+*
+* @author  Argyro Gounari
+* @version 1.0
+* @since   12-06-2018 
+*/
+public class Dictionary {
+
+	String dictionary[] = {"CAR", "CARD", "CART", "CAT"};
```

Comment:
Writing String[] = dictionary instead of this way around makes it clear that the type of dictionary is a string array.